build.gradle

```diff
@@ -0,0 +1,292 @@
+buildscript {
+  repositories {
+    maven {
+      url 'https://plugins.gradle.org/m2/'
+    }
+  }
+  dependencies {
+    // check updates with https://plugins.gradle.org/
+    classpath 'io.spring.gradle:dependency-management-plugin:1.0.4.RELEASE'
```

Comment:
I suggest to order the list by alphabetical order (and to comment this ordering).